types/underscore/index.d.ts

```diff
@@ -5327,48 +5327,48 @@ declare module _ {
         * Wrapped type `any[]`.
         * @see _.indexBy
         **/
-        indexBy(iterator: _.ListIterator<T, any>, context?: any): _Chain<T>;
+        indexBy(iterator: _.ListIterator<T, any>, context?: any): _Chain<T, Dictionary<T>>;
 
         /**
         * Wrapped type `any[]`.
         * @see _.indexBy
         **/
-        indexBy(iterator: string, context?: any): _Chain<T>;
+        indexBy(iterator: string, context?: any): _Chain<T, Dictionary<T>>;
 
         /**
         * Wrapped type `any[]`.
         * @see _.countBy
         **/
-        countBy(iterator?: _.ListIterator<T, any>, context?: any): _Chain<T>;
+        countBy(iterator?: _.ListIterator<T, any>, context?: any): _Chain<number, Dictionary<number>>;
 
         /**
         * Wrapped type `any[]`.
         * @see _.countBy
         **/
-        countBy(iterator: string, context?: any): _Chain<T>;
+        countBy(iterator: string, context?: any): _Chain<number, Dictionary<number>>;
 
         /**
         * Wrapped type `any[]`.
         * @see _.shuffle
         **/
-        shuffle(): _Chain<T>;
+        shuffle(): _Chain<T, T[]>;
 
         /**
         * Wrapped type `any[]`.
         * @see _.sample
         **/
-        sample<T>(n: number): _Chain<T>;
+        sample<T>(n: number): _Chain<T, T[]>;
 
         /**
         * @see _.sample
         **/
-        sample<T>(): _Chain<T>;
+        sample<T>(): _Chain<T, T[]>;
```

Comment:
this actually retuns a single value - please use _ChainSingle

Reply:
Done.